usr_bin/grml-resolution

```diff
@@ -27,34 +27,35 @@ CURRENT_NUM=$(xrandr | awk '/\*/ {print $1}' | tr -d '*')
 CURRENT_RESOLUTION=$(xrandr | awk '/\*/ {print $2 $3 $4}')
 
 # menu
-for i in $(xrandr | awk {'print $2$3$4'} | grep "^[0-9]") ; do
+for i in $(xrandr | awk '{print $2$3$4}' | grep "^[0-9]") ; do
   STRING="$STRING $COUNTER $i"
   ((COUNTER++))
 done
 
 # Menu Tool
-dialog --title "$PN" --menu "Change X resolution via xrandr (current resolution: $CURRENT_RESOLUTION):" 0 0 0 $STRING 2>$RESOLUTION
+dialog --title "$PN" --menu "Change X resolution via xrandr (current resolution: $CURRENT_RESOLUTION):" 0 0 0 "$STRING" 2>"$RESOLUTION"
```

Comment:
This doesn't work and causes the script to immediately exit with `ESC pressed` when getting invoked.